components/feature/downloads/src/main/java/mozilla/components/feature/downloads/AbstractFetchDownloadService.kt

```diff
@@ -411,7 +411,7 @@ abstract class AbstractFetchDownloadService : Service() {
             }
         }
 
-        private const val FILE_PROVIDER_EXTENSION = ".fileprovider"
+        private const val FILE_PROVIDER_EXTENSION = ".feature.downloads.fileprovider"
```

Comment:
This may *also* be a breaking change in Fenix? I'm not 100% sure though.

Reply:
Yes, after this PR we have to remove the provider from Fenix.

Reply:
JK, we just need to remove the `file_paths.xml` file in Fenix. Can you submit a follow up PR for that once this is merged? 😄 

Otherwise I think this will still crash in Fenix.

Reply:
Yes, I'm waiting for merging this to submit a pr for Fenix.